src/analysis.ts

```diff
@@ -213,6 +213,7 @@ export async function analyzeFolders(
   symlinksEnabled = false,
   maxPayload = MAX_PAYLOAD,
   defaultFileIgnores = IGNORES_DEFAULT,
+  sarif = false,
 ): Promise<IFileBundle> {
```

Comment:
`question`: I am surprised we don't have extend an interface here. Is it really there already or there is an opportunity for making it better?

Reply:
it's there, it's 4 levels deep 😅 
```
export interface IBundleResult {
 ...
  readonly sarifResults?: Log;
 ...
}

interface IBundleBase extends IBundleArgs, IBundleResult {}

export interface IGitBundle extends IBundleBase {
...
}

export interface IFileBundle extends IBundleBase {
...
}
```